ansible/vars/dev.yml

```diff
@@ -24,6 +24,7 @@ active_sites:
 
 deploy_env: dev
 fake_ssl_cert: yes
+ansible_user_password: 'YIN9OONyKqkxY' # blahblah
```

Comment:
heh, for some weird reason I'd be more comfortable if this said "secret" or something just so it was obviously a test password...